core/lexer.h

```diff
@@ -56,14 +56,14 @@ static inline std::ostream &operator<<(std::ostream &o, const FodderElement &f)
 {
     switch (f.kind) {
         case FodderElement::LINE_END:
-        o << "END(" << f.blanks << ", " << f.indent << ", " << f.comment[0] << ")";
-        break;
+            o << "END(" << f.blanks << ", " << f.indent << ", " << f.comment[0] << ")";
```

Comment:
I have a moderate preference for keeping them on the same line, because they're essentially just indirect goto labels.  However if there's no supported option in clang for that then I can live with it.